__init__.py

```diff
@@ -13,13 +13,16 @@
 # limitations under the License.
 import re
 import sys
+import ddg3 as ddg
```

Comment:
re-order the imports to be:
python built-ins
blank line
third party libraries
blank line
mycroft imports

Reply:
I did this but I also recall a previous comment from you on another PR of mine stating absolute imports first, relative imports second so there's that.